frontend/nepali-language-explorer/style.css

```
@@ -0,0 +1,877 @@
+    --bg-main: #0b1120;
+    --bg-secondary: #131d33;
+    --bg-card: rgba(22, 33, 58, 0.75);
+    --bg-card-hover: rgba(30, 45, 78, 0.9);
+    --border-color: rgba(255, 255, 255, 0.12);
+    --border-accent: rgba(245, 158, 11, 0.35);
+    
+    --text-primary: #f8fafc;
+    --text-secondary: #cbd5e1;
+    --text-muted: #94a3b8;
+    
+    --font-heading: 'Playfair Display', Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)